packages/gqlpt/package.json

```diff
@@ -8,7 +8,7 @@
     "build"
   ],
   "scripts": {
-    "test": "NODE_ENV=test jest",
+    "test": "cross-env NODE_ENV=test jest",
```

Comment:
We need to have `cross-env` in the dependencies for this to work on all machines, it might be working as you and the CI have it globally installed.